test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -290,15 +290,47 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 	})
 
 	Context("RPS configuration", func() {
-		It("[test_id:55012] Should have the correct RPS configuration", func() {
+		BeforeEach(func() {
 			if profile.Spec.CPU == nil || profile.Spec.CPU.Reserved == nil {
 				Skip("Test Skipped due nil Reserved cpus")
 			}
+		})
+		It("[test_id: 59572] Check RPS Mask is applied to atleast one single rx queues of a veth interface", func() {
+			if profile.Spec.WorkloadHints != nil && profile.Spec.WorkloadHints.RealTime != nil &&
+				!*profile.Spec.WorkloadHints.RealTime && !profileutil.IsRpsEnabled(profile) {
+				Skip("realTime Workload Hints is not enabled")
+			}
+			expectedRPSCPUs, err := cpuset.Parse(string(*profile.Spec.CPU.Reserved))
+			Expect(err).ToNot(HaveOccurred())
+			for _, node := range workerRTNodes {
+				//Get one veth interface
+				count := 0
+				cmd := []string{"/bin/sh", "-c", fmt.Sprintf("ip -c link show type veth | head -n 1")}
+				vethInterfaceOutput, err := nodes.ExecCommandOnNode(cmd, &node)
+				Expect(err).ToNot(HaveOccurred())
+				vethParts := strings.Split(vethInterfaceOutput, " ")
+				output := strings.Split(vethParts[1], "@")[0]
+				// we need to ensure there are no escape characters
```

Comment:
from where the escape characters can come from?

Reply:
the `ip`  command output

Reply:
Some times i get the escape characters "\x1b[36m" in the ip command output

Reply:
@Tal-or @ffromani I have used the /sys/class/net instead of ip command. Please check the latest commit.